services/queue/schemas/v1/task-group-sealed.yml

```diff
@@ -0,0 +1,38 @@
+$schema: "/schemas/common/metaschema.json#"
+title:              "Task Group Sealed Message"
+description: |
+  Message written once a task group has been sealed.
+type:               object
+properties:
+  version:    {$const: message-version}
+  taskGroupId:
+    title:          "Task-Group Identifier"
+    description: |
+      Identifier for the task-group being listed.
+    type:           string
+    pattern:        {$const: slugid-pattern}
+  schedulerId:
+    title:          "Scheduler Identifier"
+    description: |
+      Identifier for the scheduler that created this task-group.
+    type:           string
+    minLength:      {$const: identifier-min-length}
+    maxLength:      {$const: identifier-max-length}
+    pattern:        {$const: identifier-pattern}
+  sealed:
+    type:           string
+    format:         date-time
+    title:          "Sealing timestamp"
+    description: |
+      Date and time when task group was sealed.
+  expires:
+    type:           string
+    format:         date-time
+    title:          "Task group expiry date"
+    description: |
+      Date and time when task group expires.
+required:
+ - taskGroupId
+ - schedulerId
+ - expires
+additionalProperties: false
```

Comment:
I don't think this whole file is needed, because task-group-response and task-group-sealed should be the same schema. Can you merge them into one, so we just need to maintain this schema in one place?

Reply:
Ah. this is actually a message schema that is being published to the pulse queue

Reply:
Rest of the code is doing the same - separating queue schemas and api

Reply:
There is no reason to duplicate pulse schemas (or any other). See for example:

* https://github.com/taskcluster/taskcluster/blob/main/services/hooks/src/exchanges.js#L58-L107

Here we have three different hooks exchanges all using the same schema: `"pulse-hook-changed-message.json#"`.

If you've seen this duplicated elsewhere, can you create an issue for it? We should avoid the code duplication as much as possible, but it has a tendency to slip in. Thanks Yarik!

Reply:
Yarik has pointed out to me that one is an http api schema, and the other is a schema for a pulse message, and indeed, we tend not to share these, so let's leave them as separate after all, apologies for the noise.
I think the only thing we might want to change is rename:

* task-group-sealed.yml -> task-group-sealed-message.yml
* task-group-resolved.yml -> task-group-resolved-message.yml

just to be consistent with the naming scheme we are using elsewhere.

We can probably review elsewhere (in a different issue/PR) if we want to start sharing schemas between http api input/outputs and pulse messages, or if it makes sense to keep them separate.

Thanks Yarik!